### ISSUE TITLE
NewLine missing between Usage and Example

### ISSUE THREAD

**Issue by msevestre** [commandlineparser/commandline#185]
Hi there, 

I am using 2.1.1 beta. 
First this is a fantastic and easy to use library. Thanks a lot

I have just implemented some `Usage` feature according to documentation found in Wiki. Your example shows some newline between the Usage section and the Error section (see [here](https://github.com/commandlineparser/commandline/wiki#usage-attribute)). However they do not show up on my tests.

See output
```
PKSim
Copyright c 2017 - Open Systems Pharmacology Community

ERROR(S):
  Required option 'i, input' is missing.
  Required option 'o, output' is missing.
USAGE:
Create snapshot files from project files:
  PKSim.CLI snap --input <SnapshotFolder> --output <ProjectFolder> --snapshot
Create project files from snapshot files:
  PKSim.CLI snap --input <ProjectFolder> --output <SnapshotFolder> --project
Create project files from snapshot files and only log errors:
  PKSim.CLI snap --input <ProjectFolder> --logLevel Error --output <SnapshotFolder> --project
```

Would it be possible to add a Newline before `USAGE`. Also maybe a new line between each Example?

Cheers,
Michael

**Comment by ericnewton76** [comment]
Hi, how exactly are you generating that?  Can you point us to the code in your repo or copy the relevant code here?

I'll check it out and see what can be done.

**Comment by msevestre** [comment]
Hi @ericnewton76 

You can find the code [here](https://github.com/Open-Systems-Pharmacology/PK-Sim/blob/develop/src/PKSim.CLI/Commands/SnapshotRunCommand.cs
)

Let me know if you need more info,

**Comment by DominikWiesend** [comment]
waiting for this too, I thought already I`m to stupid. Feels good to see another one stuck there.

**Comment by msevestre** [comment]
@ericnewton76 Any news on that?

**Comment by mchenzero** [comment]
Also I think there should be a line break before the `USAGE:` heading.

**Comment by segunak** [comment]
This issue is addressed by #494
